x/programs/rust/examples/single_nft/src/example.rs

```diff
@@ -0,0 +1,133 @@
+use wasmlanche_sdk::simulator::{
+    Client, Endpoint, ExecuteResponse, Key, Param, ParamType, Plan, Step,
+};
+
+// TODO: remove when the simulator merges
+#[allow(dead_code)]
+const PROGRAM_ID: &str = "0000000000000000000000000000000000000000000000001";
+
+// TODO: remove when the simulator merges
+#[allow(dead_code)]
+fn initialize_plan<'a>(
+    nft_name: &'a str,
+    nft_name_length: &'a str,
+    nft_symbol: &'a str,
+    nft_symbol_length: &'a str,
+    nft_uri: &'a str,
+    nft_uri_length: &'a str,
```

Comment:
The lifetime doesn't need to be specified here. The rule of thumb is that if there are no lifetime bounds in the return type, there doesn't need to be any lifetime bounds in the parameters.